inference-engine/src/inference_engine/include/openvino/runtime/remote_context.hpp

```diff
@@ -134,7 +135,7 @@ class OPENVINO_RUNTIME_API RemoteContext {
      * Abstract method.
      * @return A map of name/parameter elements.
      */
-    ParamMap get_params() const;
+    Parameters get_params() const;
```

Comment:
Can we use sharedAny?

Reply:
No: https://github.com/openvinotoolkit/openvino/pull/7750#discussion_r721157057